StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/SavedPaymentMethods/SavedPaymentMethodsSheetError.swift

```diff
@@ -8,31 +8,8 @@ import Foundation
 import StripePayments
 
 @_spi(PrivateBetaSavedPaymentMethodsSheet) public enum SavedPaymentMethodsSheetError: Error {
-
     /// Error while fetching saved payment methods attached to a customer
     case errorFetchingSavedPaymentMethods(Error)
```

Comment:
Just checking my understanding -- these are errors for terminal states of the sheet closing -- so, there's really only one case -- where we would fail to fetch the saved Payment methods, and then close the sheet?

Reply:
That's the only terminal error state I can think of atm. I need to try to brainstorm a few more.